server/src/models/load_utils.js

```diff
@@ -42,6 +42,8 @@ const bilingual_remap = (obj, dest_key, source_key) => ({
   [`${dest_key}_fr`]: obj ? obj[`${source_key}_fr`] : null,
 });
 
+const check_ngtive_nan = (value) => _.isNaN(+value) ? null : +value;
```

Comment:
Can you explain this one to me? I don't think what it's doing lines up with its name, but I'm also curious why it's needed for `orgTransferPayments` and `orgVoteStatPa` and not for any other tables. Is there something weird with the data in those two csvs?

Reply:
From what I recall, this was because of negative values and NaNs. populate db won't cast value to number because of '-'.. Couldn't replicate the NaN issue though

ValidationError: OrgTransferPayments validation failed: pa_last_year_5_auth: Cast to Number failed for value "-" at path "pa_last_year_5_auth"

Reply:
Ah, ok. It's a bit confusing because the function name here is the "why", when we usually name things after the "what".

Looking around, something like this is done in other populate scripts that are handling numbers too (well, at least line 54 of `result/populate.js`). We probably want something like this in `load_utils.js`
```
// Populate time validation will choke on either the string representation of a negative number or a value of NaN
// Need to parse strings to numbers ourselves to handle negatives, but take care to preserve nulls which we assume are csv blanks (trying to convert a null gives a NaN)
// Preserving nulls instead of defaulting to nulls because we still want validation errors to surface when any other data issue produces a NaN!
const null_preserving_to_number = (value) => _.isNull(value) ? _.toNumber(value) : null;
```

Could you use that and do a quick look in the other populate scripts for things following a similar pattern? Thanks!

Reply:
So I think the problems are
1. if value isn't `null` (meaning value has actual numbers), this will always be null
2. It has to use _isNaN because some values are "-". isNull and isNaN will have different output for that.

So I rewrote it using _.toNumber but still not sure if this is sufficient. Please have a look..!!

Reply:
My bad on two counts
  1. was missing a `!` before the `_.isNull` in the example code above, just a typo
  2. ahh, wait, I thought you were saying that any string containing a '-' was failing to validate as a number with mongo/mongoose, e.g. negatives (maybe I was confused by the original function name mentioning negatives). I hadn't realized that there are some cells in the csvs that contain only a '-'. 

Those cells that contain only a '-' look to me like data errors, in which case my version (without the typo) does what I wanted by surfacing errors instead of suppressing them! We want to know when a non-number is in a numeric column of a csv because that's a sign something's broken up stream.

I'll talk to the pipeline folks about those -'s. The proper fix here is probably to get cleaner data, not to code around the issues.

Reply:
Yup, those '-' cells should probably just be empties, Ross will confirm and fix them on the pipeline side.

Reply:
Okay! so then I'll wait until it's fixed on pipeline side and see if I can remove this entirely, or go with your approach?

Reply:
I think my approach, corrected, and replace all the "-" cells in org_transfer_payments.csv with 0's on this branch for now so you can continue testing. We'll just wait for the pipeline's final fix before merging this.

Reply:
Ok!

Reply:
Data updated on master, -'s should all be zeros now.